### PR TITLE
[JBWS-4196] Correctly rewrite the WSDL address when server only has a HTTPS listener

### PR DESCRIPTION
**Pull request by iweiss**
Issue: https://issues.jboss.org/browse/JBWS-4196

This is related to https://issues.jboss.org/browse/WFLY-12135

### CODE REVIEW

**Comment by iweiss** [comment]
@asoldano @jimma can you please review this PR?

File: src/main/java/org/jboss/ws/common/deployment/EndpointAddressDeploymentAspect.java [hunk 1]
```diff
@@ -127,16 +127,20 @@ public void start(Deployment dep)
...
          return AbstractServerConfig.getServerIntegrationServerConfig();
       }
       return AccessController.doPrivileged(AbstractServerConfig.GET_SERVER_INTEGRATION_SERVER_CONFIG);
    }
-   
+
    protected boolean isConfidentialTransportGuarantee(final Deployment dep, final Endpoint ep)
    {
+      // Fix for JBWS-4196/WFLY-12135. We pick up the attached boolean. If it's true then isConfidentialTransportGuarantee is true
+      boolean isHttps = dep.getAttachment(Boolean.class);
```
Comment (jimma) [inline src/main/java/org/jboss/ws/common/deployment/EndpointAddressDeploymentAspect.java:141]: Is dep.getProperty("isHttps") better ?

Comment (iweiss) [reply src/main/java/org/jboss/ws/common/deployment/EndpointAddressDeploymentAspect.java:141]: I think we can. Let me try.

**Comment by iweiss** [comment]
It seems something is broken on Travis.

**Pull request merged**